Bio/PDB/__init__.py

```diff
@@ -10,7 +10,16 @@
 Includes: PDB and mmCIF parsers, a Structure class, a module to keep a local
 copy of the PDB up-to-date, selective IO of PDB files, etc.
 
-Author: Thomas Hamelryck.  Additional code by Kristian Rother.
+Original Author: Thomas Hamelryck.
+Contributions by:
+- Peter Cock
+- Joe Greener
+- Rob Miller
+- Lenna X. Peterson
+- Joao Rodrigues
+- Kristian Rother
+- Eric Talevich
+- and many others.
```

Comment:
I think having module contributors other than the original author(s) listed in the docstring is unusual for Biopython (usually author and major contributors' names are in the copyright block at the top of each file). Although that doesn't mean we can't change this...